bylaws.md

```diff
@@ -168,8 +159,13 @@ All members must provide and monitor an email address which will serve as the so
 Emails from the board must be sent from an @texaslan.org (http://texaslan.org/) email account.
 Any communications through social media must follow an official announcement in email form.
 
-**Article X. Amendments**
+**Article X. Board Orders**
+
+Processes that aren't required to be in the Bylaws shall be placed in the [Board Orders Document](./board_orders.md), to where it defines extra processes these bylaws must follow.
+These Board Orders can be altered by a vote of 2/3rds of the Board.
+
+**Article XI. Amendments**
 
 Changes to these bylaws may proposed and drafted by the Board, which is will then present the proposed changes to the general membership in a full meeting called for that purpose.
-Additionally, once every academic year in the spring semester, the current Board shall meet with the general membership to discuss the current bylaws and determine if any changes must be made.
+Additionally, once every academic year in the spring semester, the current Board shall meet with the general membership to discuss the current bylaws and board orders, and determine if any changes must be made.
```

Comment:
Capitalize all instances of an introduced and defined term. In this case: "Bylaws and Board Orders"